### DISCUSSION TITLE
The project is coming to an end

### DISCUSSION THREAD

**Discussion by langroodi** [langroodi/Adaptive-AUTOSAR#21] [Announcements]
Dear Adaptive AUTOSAR simulator community,

This project is coming to and end. Almost all the planned scope has been implemented and just the small part of that has been skipped due to lacking enough time for development.

From this point, no new feature will be added to the simulator and no existing part will be updated to the latest ARA standard. However, some code cleaning and enhancement may be performed in future and bug reporting still be welcomed and will be considered (if it won't take too much fixing effort). You are still free to ask questions at the [Discussion board](https://github.com/langroodi/Adaptive-AUTOSAR/discussions), and I will try to answer them as much as my knowledge/memory let me to do so (please do not open issues for your questions).

There is an online event for people who are interested in the development journey of this project (from the beginning to the end) and also for those who want to fork the project and add new cool stuffs to it and they need some help/insight. So, feel to post your questions here and I will try to answer them in that event.

Cheers,
_Armin_

**Comment by duongtt1** [comment]
Respect you 👍👍👍

**Comment by langroodi** [comment]
The project wrap up online event will be on August 11th 4:00 PM CEST which is accessible from the following link. I will try to record the event and upload it to my Youtube as well, then I will share that link here too:

https://www.linkedin.com/events/adaptiveautosarlinuxsimulator7091032043492372481/theater/

**Reply by langroodi** [reply]
The event recording on YouTube: https://youtu.be/OFYA-raVrWE
